src/applications/caregivers/tests/unit/components/FormPages/FacilityConfirmation.unit.spec.jsx

```diff
@@ -47,65 +46,75 @@ describe('CG <FacilityConfirmation>', () => {
+      expect(queryByText(new RegExp(selectedFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.exist;
+      expect(queryByText(new RegExp(address.address2))).to.exist;
+      expect(queryByText(new RegExp(address.address3))).to.exist;
     });
 
-    it('calls goBack callback on click', () => {
+    it('should render the assigned support facility name and address when data is provided', () => {
+      const { queryByText } = subject();
+      const address = caregiverFacility.address.physical;
+      expect(queryByText(new RegExp(caregiverFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.exist;
+      expect(queryByText(new RegExp(address.address2))).to.exist;
+      expect(queryByText(new RegExp(address.address3))).to.exist;
+    });
+
+    it('should render just the facility name when address is omitted from facility data', () => {
+      const { queryByText } = subject({
+        assigned: { ...caregiverFacility, address: undefined },
+      });
+      const address = caregiverFacility.address.physical;
+      expect(queryByText(new RegExp(caregiverFacility.name))).to.exist;
+      expect(queryByText(new RegExp(address.address1))).to.not.exist;
+      expect(queryByText(new RegExp(address.address2))).to.not.exist;
+      expect(queryByText(new RegExp(address.address3))).to.not.exist;
+    });
+
+    it('should call `goBack` when the Back button is clicked', () => {
       const { selectors } = subject();
-      userEvent.click(selectors().formNavButtons.back);
+      userEvent.click(selectors().backBtn);
       expect(goBack.calledOnce).to.be.true;
     });
 
-    it('calls goForward callback on click', () => {
+    it('should call `goForward` when the Continue button is clicked', () => {
       const { selectors } = subject();
-      userEvent.click(selectors().formNavButtons.forward);
+      userEvent.click(selectors().continueBtn);
       expect(goForward.calledOnce).to.be.true;
     });
+  });
 
-    context('review mode', () => {
-      beforeEach(() => {
-        global.window.location = { search: '?review=true' };
-      });
-
-      it('calls goToPath to facility search on back click', () => {
-        const { selectors } = subject();
-        userEvent.click(selectors().formNavButtons.back);
-        expect(
-          goToPath.calledWith(
-            '/veteran-information/va-medical-center/locator?review=true',
-          ),
-        ).to.be.true;
-      });
-
-      it('calls goToPath callback to review page on forward click', () => {
-        const { selectors } = subject();
-        userEvent.click(selectors().formNavButtons.forward);
-        expect(goToPath.calledWith('/review-and-submit')).to.be.true;
+  context('when the page renders in review mode', () => {
+    beforeEach(() => {
+      Object.defineProperty(window, 'location', {
+        configurable: true,
+        value: { search: '?review=true' },
       });
```

Comment:
This replaces the previous method of setting `global.window.location` to value of `{ search: '?review=true' }`. Modifying the `global` object in the test suite is a no-no.